extern/filereaderlp/reader.cpp

```diff
@@ -892,7 +892,12 @@ void Reader::readnexttoken(bool& done) {
    done = false;
 
    if (this->linebufferpos == this->linebuffer.size()) {
-     // read next line
+     // read next line if any are left. 
+     if (this->file.eof()) {
+         this->rawtokens.push_back(std::unique_ptr<RawToken>(new RawToken(RawTokenType::FLEND)));
+         done = true;

```

Comment:
The value of `done` is never actually used for anything; I can remove that in a separate PR if relevant.

Reply:
Thanks, but let's leave that to @feldmeier , who might want it left in for future use